client/src/components/Button/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import {
+  faArrowLeft,
+  faArrowRight,
+  faCheckCircle
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+import './style.css'
+
+export default function Button(props) {
+  const { label, style, done, rightIcon, leftIcon, onClick } = props;
+  return (
+        <button className={`button ${props.className}`} onClick={onClick}>
+          {leftIcon? <FontAwesomeIcon icon={faArrowLeft} className=''/>: null}
+            <span className='button-title'>{label}</span>
```

Comment:
indentation !! 
why is the span looks like it's nested in but it's not?

Reply:
Done